gatsby/installing.rst

```diff
@@ -35,9 +35,11 @@ The CLI allows to initialize a project:
 This command says gatsby-cli to create a new Gatsby project with name `hello-world` with a basic default file structure.
 There are several boilerplates created by the community that allows to easily bootstrap an application for different use-cases.
 
-Boilerplates are called `starters` and in the `offical site<https://www.gatsbyjs.org/docs/gatsby-starters/>`_ you could
+These boilerplates are called `starters` and in the `offical site<https://www.gatsbyjs.org/docs/gatsby-starters/>`_ you could
```

Comment:
s/`starters`/"starters"

Add a space between labels and targets for links.
```rst
`offical site <https...
```

Reply:
Fix spelling: "official"

Still missing that space between `e<`. This will not render correctly.